browser/main/Main.js

```diff
@@ -144,6 +152,28 @@ class Main extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    this.setState({ fullScreen: !this.state.fullScreen }, () => {
+      const noteDetail = document.querySelector('.NoteDetail')
+      const mainBody = document.querySelector('#main-body')
+      const sliderRight = document.querySelector('#slider-right')
+      const sliderLeft = document.querySelector('#slider-left')
+      if (this.state.fullScreen) {
+        this.state.noteDetailWidth = noteDetail.style.left
+        this.state.mainBodyWidth = mainBody.style.left
+        noteDetail.style.left = '0px'
+        mainBody.style.left = '0px'
+        sliderRight.style.display = 'none'
+        sliderLeft.style.display = 'none'
+      } else {
+        noteDetail.style.left = this.state.noteDetailWidth
+        mainBody.style.left = this.state.mainBodyWidth
+        sliderRight.style.display = 'block'
+        sliderLeft.style.display = 'block'
+      }
+    })
+  }
```

Comment:
I cannot reproduce the behavior. Could you tell me more detail?
![7402fd6264e5b2d1d41a5df2268b813b](https://cloud.githubusercontent.com/assets/11307908/26611721/004c05bc-45eb-11e7-8ce5-022282f22a17.gif)

How about `showListColumns()` and `hideListColumns()`?